openlibrary/templates/type/edition/view.html

```diff
@@ -407,10 +407,10 @@ <h3 class="header">
           $if contributors:
             <div class="section">
               <h3 class="collapse">$_("Contributors")</h3>
-              <ul class="contributors">
-                $for c in contributors:
-                  $:display_contributor(c)
```

Comment:
Nothing in this template appears to be using `display_contributor` now so the macro should be removed.

https://github.com/internetarchive/openlibrary/pull/4373/files#diff-93ec72c37de495b619f24641f2b9defc4ea1205bb5450bb6baa33498831721efR126

Reply:
@jdlrobson done